autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleScalingActivityV1Controller.java

```diff
@@ -0,0 +1,106 @@
+package com.sequenceiq.periscope.controller;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Controller;
+
+import com.sequenceiq.authorization.annotation.CheckPermissionByResourceCrn;
+import com.sequenceiq.authorization.annotation.CheckPermissionByResourceName;
+import com.sequenceiq.authorization.annotation.ResourceCrn;
+import com.sequenceiq.authorization.annotation.ResourceName;
+import com.sequenceiq.authorization.resource.AuthorizationResourceAction;
+import com.sequenceiq.periscope.api.endpoint.v1.DistroXAutoScaleScalingActivityV1Endpoint;
+import com.sequenceiq.periscope.api.model.DistroXAutoscaleScalingActivityResponse;
+import com.sequenceiq.periscope.converter.DistroXAutoscaleScalingActivityResponseConverter;
+import com.sequenceiq.periscope.service.ScalingActivityService;
+
+@Controller
+public class DistroXAutoScaleScalingActivityV1Controller implements DistroXAutoScaleScalingActivityV1Endpoint {
+
+    @Inject
+    private DistroXAutoscaleScalingActivityResponseConverter distroXAutoscaleScalingActivityResponseConverter;
+
+    @Inject
+    private ScalingActivityService scalingActivityService;
+
+    @Override
+    @CheckPermissionByResourceName(action = AuthorizationResourceAction.SCALE_DATAHUB)
```

Comment:
Are you sure that all the authz annotations added are working as intended? Is this tested with real-UMS? If I'm not wrong, proper authz checks require implementing `AuthorizationResourceCrnProvider` and `AuthorizationEnvironmentCrnProvider`, in the service layer. See `AutoScaleClusterCommonService` https://github.com/hortonworks/cloudbreak/blob/e16ff1c96f9339a58644bf7cac029b7db9195621/autoscale/src/main/java/com/sequenceiq/periscope/controller/AutoScaleClusterCommonService.java#L46

Reply:
Done